modules/multi-architecture-modify-machine-set-aws.adoc

```diff
@@ -97,11 +97,11 @@ spec:
 +
 [source,terminal]
 ----
-$ oc get -o jsonpath=‘{.status.infrastructureName}{“\n”}’ infrastructure cluster
+$ oc get -o jsonpath="{.status.infrastructureName}{'\n'}" infrastructure cluster
 ----
 <2> Specify the infrastructure ID, role node label, and zone.
```

Comment:
```suggestion
<2> Specifies the infrastructure ID, role node label, and zone.
```

Reply:
Start the sentence with "Specifies" instead of specify. You can change it wherever it applies.
See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks

Reply:
Same comment. 

Using "Specifies" is not required here. We are asking users to specify their value for a given placeholder.